examples/operator-rbac/1-setup-operator-rbac.ipynb

```diff
@@ -0,0 +1,753 @@
+      "<jemalloc>: (This is the expected behaviour if you are running under QEMU)\r\n",
+      "/opt/app-root/lib64/python3.11/site-packages/feast/feature_view.py:48: DeprecationWarning: Entity value_type will be mandatory in the next release. Please specify a value_type for entity '__dummy'.\r\n",
+      "  DUMMY_ENTITY = Entity(\r\n",
+      "/opt/app-root/lib64/python3.11/site-packages/pydantic/_internal/_fields.py:192: UserWarning: Field name \"vector_enabled\" in \"SqliteOnlineStoreConfig\" shadows an attribute in parent \"VectorStoreConfig\"\r\n",
+      "  warnings.warn(\r\n",
+      "/opt/app-root/lib64/python3.11/site-packages/pydantic/_internal/_fields.py:192: UserWarning: Field name \"vector_len\" in \"SqliteOnlineStoreConfig\" shadows an attribute in parent \"VectorStoreConfig\"\r\n",
+      "  warnings.warn(\r\n",
+      "/feast-data/feast_rbac/feature_repo/example_repo.py:27: DeprecationWarning: Entity value_type will be mandatory in the next release. Please specify a value_type for entity 'driver'.\r\n",
+      "  driver = Entity(name=\"driver\", join_keys=[\"driver_id\"])\r\n",
+      "Applying changes for project feast_rbac\r\n",
+      "/opt/app-root/lib64/python3.11/site-packages/feast/feature_store.py:579: RuntimeWarning: On demand feature view is an experimental feature. This API is stable, but the functionality does not scale well for offline retrieval\r\n",
+      "  warnings.warn(\r\n",
+      "Created project \u001B[1m\u001B[32mfeast_rbac\u001B[0m\r\n",
+      "Created entity \u001B[1m\u001B[32mdriver\u001B[0m\r\n",
+      "Created feature view \u001B[1m\u001B[32mdriver_hourly_stats_fresh\u001B[0m\r\n",
+      "Created feature view \u001B[1m\u001B[32mdriver_hourly_stats\u001B[0m\r\n",
+      "Created on demand feature view \u001B[1m\u001B[32mtransformed_conv_rate_fresh\u001B[0m\r\n",
+      "Created on demand feature view \u001B[1m\u001B[32mtransformed_conv_rate\u001B[0m\r\n",
+      "Created feature service \u001B[1m\u001B[32mdriver_activity_v1\u001B[0m\r\n",
+      "Created feature service \u001B[1m\u001B[32mdriver_activity_v3\u001B[0m\r\n",
+      "Created feature service \u001B[1m\u001B[32mdriver_activity_v2\u001B[0m\r\n",
+      "Created permission \u001B[1m\u001B[32mfeast_admin_permission\u001B[0m\r\n",
+      "Created permission \u001B[1m\u001B[32mfeast_user_permission\u001B[0m\r\n",
+      "\r\n",
+      "Created sqlite table \u001B[1m\u001B[32mfeast_rbac_driver_hourly_stats_fresh\u001B[0m\r\n",
+      "Created sqlite table \u001B[1m\u001B[32mfeast_rbac_driver_hourly_stats\u001B[0m\r\n",
+      "\r\n"
+     ]
+    }
+   ],
+   "execution_count": 162
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": "**List the applied permission details permissions on Feast Resources.**"
+  },
+  {
+   "cell_type": "code",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2025-03-05T18:51:01.464386Z",
+     "start_time": "2025-03-05T18:50:19.362018Z"
+    }
+   },
+   "source": [
```

Comment:
The same could be demonstrated by connecting a client to the registry server, w/o using the `kubectl exec` command. e.g., probably in the client notebook.
It probably depends on who's the target person we want to instruct with this notebook.

Reply:
This is for Feast Admin part for setting up permission and then to validate applied permissions. Yup can be done with client and then connecting registry also I believe using `CLI`, if required.